src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -29,7 +29,7 @@ class DebugAccessDecisionManager implements AccessDecisionManagerInterface
     private $voters;
     private $decisionLog = array();
 
-    public function __construct(AccessDecisionManager $manager)
+    public function __construct(AccessDecisionManagerInterface $manager)
```

Comment:
the retrieval of the strategy with reflection below should check for being an instance of AccessDecisionManager though (btw, the ReflectionProperty should use ``AccessDecisionManager::class`` as first argument to be compatible with inheritance, as this is a private property which is not accessible on a child class)